content/version/2/1/code_of_conduct.id.md

```diff
@@ -0,0 +1,140 @@
++++
+version = "2.1"
+aliases = ["/version/2/1/id"]
++++
+
+# Kode Etik Contributor Covenant
+
+## Ikrar Kami
+
+Kami sebagai anggota, kontributor, dan pengelola berikrar untuk membuat
+partisipasi dalam komunitas kami terbebas dari pelecehan bagi semua orang, tanpa
+memandang usia, ukuran tubuh, kecacatan yang tampak maupun tidak, kelompok
+etnis, karakteristik seksual, identitas gender dan ekspresinya, tingkat
+pengalaman, pendidikan, status sosial ekonomi, kewarganegaraan, penampilan
+pribadi, ras, kasta, warna kulit, agama, atau identitas dan orientasi seksual.
+
+Kami berjanji untuk bersikap dan berinteraksi dalam langkah komunitas yang
+terbuka, ramah, beragam, merangkul, dan sehat.
+
+## Standar Kami
+
+Contoh perilaku yang berkontribusi pada lingkungan positif dalam komunitas kami
+meliputi:
+
+* Menunjukkan rasa empati dan kebaikan kepada orang lain
+* Menghormati pendapat, pandangan, dan pengalaman yang berbeda
+* Memberikan dan menerima secara sopan umpan balik yang membangun
+* Menerima tanggung jawab dan meminta maaf kepada yang terdampak dari kesalahan
+  kita, dan belajar dari pengalaman
+* Fokus terhadap apa yang terbaik tidak hanya untuk diri pribadi, tetapi untuk
+  keseluruhan komunitas
+
+Contoh perilaku yang tidak dapat diterima meliputi:
+
+* Penggunaan bahasa atau citra seksual, dan sorotan bernada seksual atau rayuan
+  dalam bentuk apapun
+* Mengejek, mencaci maki atau komentar yang menghina / merendahkan, dan serangan
+  terhadap individual atau politik
+* Pelecehan secara umum atau pribadi
+* Memuat informasi pribadi orang lain, seperti alamat fisik atau surel, tanpa
+  persetujuan orang tersebut
```

Comment:
```suggestion
* Menyebarkan informasi pribadi orang lain, seperti alamat fisik atau surel,
  tanpa persetujuan orang tersebut
```
Based on the original context, _menyebarkan_ is more related to than _memuat_, as _memuat_ has nuance to "to load/include/add" than "to share"